schema/equals.go

```diff
@@ -17,7 +17,11 @@ limitations under the License.
 package schema
 
 // Equals returns true iff the two Schemas are equal.
-func (a Schema) Equals(b Schema) bool {
+func (a *Schema) Equals(b *Schema) bool {
+	if b == nil {
```

Comment:
what if a is nil too? ;-)

maybe ...
```
if a == nil || b == nil {
  return a == nil && b == nil
}
```

Reply:
Right, i mixed up the nil behavior in my head and thought a couldn't be nil here because then calling the method would be a nil pointer. But Schema isn't an interface...